### PR TITLE
chore: Verify protoc versions in samples and archetypes

### PR DESCRIPTION
**Pull request by johanandren**
Follow up to #822 

Ping @raboof

### CODE REVIEW

**Comment by johanandren** [comment]
As expected, failing with a sample of the validation:

```
Not all Java Samples an Archetypes have the same protoc version:
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.9.1:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
                            <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${D}{os.detected.classifier}</protocArtifact>
              <protocArtifact>com.google.protobuf:protoc:3.11.4:exe:${D}{os.detected.classifier}</protocArtifact>

Exited with code exit status 1
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -146,18 +146,31 @@ jobs:
...
-              "Not all Java samples have the same SDK version:"
+              echo "Not all Java samples have the same SDK version:"
               find ./samples -name pom.xml | xargs grep "<akkaserverless-sdk.version>"
               false
             fi
+      - run:
+          name: Protoc version
+          command: |
+            AKKA_GRPC_PROTOC=$(sbt "sdkCore/protocVersion" 2>/dev/null | tail -n 1 | sed 's/\[info\] //g')
+            POM_PROTOC_VERSIONS=$(find ./samples ./maven-java/ -name pom.xml | xargs egrep -h "com.google.protobuf:protoc:" | sed 's/.*:protoc://' | sed 's/:exe.*//' | sort | uniq)
```
Comment (raboof) [inline .circleci/config.yml:165]: neato

**Review by raboof — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (octonato) [reply .circleci/config.yml:165]: were you trying to write my name or to say 'neat'?

**Review by octonato — APPROVED** [review]
This looks good, but it have annoyed me a few times in the recent days. 

I have PRs adding new samples and then each time we update the samples versions, my PRs get broken. Anyway, that's how it is. 🤷

File: .circleci/config.yml (hunk 1 above)
Comment (raboof) [reply .circleci/config.yml:165]: https://www.urbandictionary.com/define.php?term=neato - though perhaps `|) A |\/| |\|` was onto something...

Comment (johanandren) [reply .circleci/config.yml:165]: Lol. New nick name decided!

**Pull request merged**